dashboard/config/locales/scripts.en.yml

```diff
@@ -9657,7 +9657,7 @@ en:
               name: Aquatic
           title: 'Minecraft: Voyage Aquatic'
           description_audience: ''
-          description_short: ''
+          description_short: 'Minecraft is back with a brand new activity for Hour of Code!'
```

Comment:
Should there be a "the" before "Hour of Code"?  cc @tanyaparker

Reply:
I'll defer to Tanya on this -- the language isn't changing from last year, so I just copy-pasted from 2017

Reply:
Oh, yeah, I guess it is better with 'the' - can we add, Maddie?

Reply:
yep, adding!